Translations update from openSUSE Weblate

opensuse-i18n opened the pull request:
Translations update from [openSUSE Weblate](https://l10n.opensuse.org) for [Edge/Copy Edge 3.1](https://l10n.opensuse.org/projects/edge/copy-edge-3-1/).



Current translation status:

![Weblate translation status](https://l10n.opensuse.org/widget/edge/copy-edge-3-1/horizontal-auto.svg)

Commit:
Translated using Weblate (Chinese (China) (zh_CN))

Currently translated at 94.7% (3011 of 3178 strings)

Translation: Edge/Copy Edge 3.1
Translate-URL: https://l10n.opensuse.org/projects/edge/copy-edge-3-1/zh_CN/

Commit:
Translated using Weblate (Japanese)

Currently translated at 94.8% (3013 of 3178 strings)

Translation: Edge/Copy Edge 3.1
Translate-URL: https://l10n.opensuse.org/projects/edge/copy-edge-3-1/ja/

Commit:
Translated using Weblate (Korean)

Currently translated at 1.3% (42 of 3178 strings)

Translation: Edge/Copy Edge 3.1
Translate-URL: https://l10n.opensuse.org/projects/edge/copy-edge-3-1/ko/

Commit:
Translated using Weblate (Korean)

Currently translated at 1.3% (42 of 3178 strings)

Translation: Edge/Copy Edge 3.1
Translate-URL: https://l10n.opensuse.org/projects/edge/copy-edge-3-1/ko/

jfaltenbacher committed:
Translated using Weblate (Chinese (China) (zh_CN))

Currently translated at 94.7% (3011 of 3178 strings)

Translation: Edge/Copy Edge 3.1
Translate-URL: https://l10n.opensuse.org/projects/edge/copy-edge-3-1/zh_CN/

luminow committed:
Added translation using Weblate (Ukrainian)

Commit:
Translated using Weblate (Ukrainian)

Currently translated at 13.4% (429 of 3178 strings)

Translation: Edge/Copy Edge 3.1
Translate-URL: https://l10n.opensuse.org/projects/edge/copy-edge-3-1/uk/

Commit:
Translated using Weblate (Ukrainian)

Currently translated at 13.4% (429 of 3178 strings)

Translation: Edge/Copy Edge 3.1
Translate-URL: https://l10n.opensuse.org/projects/edge/copy-edge-3-1/uk/

Commit:
Translated using Weblate (Chinese (China) (zh_CN))

Currently translated at 99.8% (3174 of 3178 strings)

Translation: Edge/Copy Edge 3.1
Translate-URL: https://l10n.opensuse.org/projects/edge/copy-edge-3-1/zh_CN/

Commit:
Translated using Weblate (Japanese)

Currently translated at 99.8% (3174 of 3178 strings)

Translation: Edge/Copy Edge 3.1
Translate-URL: https://l10n.opensuse.org/projects/edge/copy-edge-3-1/ja/

Commit:
Translated using Weblate (Japanese)

Currently translated at 99.8% (3174 of 3178 strings)

Translation: Edge/Copy Edge 3.1
Translate-URL: https://l10n.opensuse.org/projects/edge/copy-edge-3-1/ja/

Commit:
Translated using Weblate (Japanese)

Currently translated at 99.8% (3174 of 3178 strings)

Translation: Edge/Copy Edge 3.1
Translate-URL: https://l10n.opensuse.org/projects/edge/copy-edge-3-1/ja/

hardys commented:
Marking this as draft after discussion with @fsundermeyer - we need to discuss the full flow and repo configuration for Weblate integration

Commit:
Translated using Weblate (Japanese)

Currently translated at 100.0% (3178 of 3178 strings)

Translation: Edge/Copy Edge 3.1
Translate-URL: https://l10n.opensuse.org/projects/edge/copy-edge-3-1/ja/

Commit:
Translated using Weblate (Chinese (China) (zh_CN))

Currently translated at 100.0% (3178 of 3178 strings)

Translation: Edge/Copy Edge 3.1
Translate-URL: https://l10n.opensuse.org/projects/edge/copy-edge-3-1/zh_CN/

luminow committed:
Translated using Weblate (Ukrainian)

Currently translated at 14.4% (458 of 3178 strings)

Translation: Edge/Copy Edge 3.1
Translate-URL: https://l10n.opensuse.org/projects/edge/copy-edge-3-1/uk/

Commit:
Translated using Weblate (Chinese (China) (zh_CN))

Currently translated at 100.0% (3178 of 3178 strings)

Translation: Edge/Copy Edge 3.1
Translate-URL: https://l10n.opensuse.org/projects/edge/copy-edge-3-1/zh_CN/

Commit:
Translated using Weblate (Japanese)

Currently translated at 100.0% (3178 of 3178 strings)

Translation: Edge/Copy Edge 3.1
Translate-URL: https://l10n.opensuse.org/projects/edge/copy-edge-3-1/ja/

luminow committed:
Translated using Weblate (Ukrainian)

Currently translated at 15.0% (478 of 3178 strings)

Translation: Edge/Copy Edge 3.1
Translate-URL: https://l10n.opensuse.org/projects/edge/copy-edge-3-1/uk/

Commit:
Translated using Weblate (Chinese (China) (zh_CN))

Currently translated at 100.0% (3178 of 3178 strings)

Translation: Edge/Copy Edge 3.1
Translate-URL: https://l10n.opensuse.org/projects/edge/copy-edge-3-1/zh_CN/
